sleap_anipose/calibration.py

```diff
@@ -61,13 +62,10 @@ def make_reproj_imgs(
             reprojected corners.
         frames: A length n_frames list of the frames visible from each view.
             These frames are used for triangulation and saved in metadata.
-        n_samples: The number of images to make per view.
         session: Path containing the view subfolders with the calibration board
-            images. Images are saved to the view subfolders in this folder as
-            'session / view / reprojection-{frame}.jpg'. Assumed to be the
-            current working directory if not specified.
-        save: Flag determining whether or not to save images to the session.
-
+            images.
+        n_samples: The number of images to make per view.
+        save_path: The session to save the images to. If not specified as a non-empty string, images will nto be saved. Images are saved to the view subfolders in this folder as 'save_path / view / reprojection-{frame}.png'.
```

Comment:
Fix typo ("nto" -> "not") and make sure line length doesn't exceed 88 chars

Reply:
resolved